test-end-to-end/android-maestro/common/copy_image_file_to_picture_folder.yaml

```diff
@@ -0,0 +1,16 @@
+appId: com.google.android.documentsui
+
+---
+
+# This flow copy an image file from downloads to pictures folder
+
+- launchApp
```

Comment:
Can't we just use the adb push command to put the picture directly into the pictures folder without needing this?